src/builtInPDBObjects/headers/TreeResult.h

```diff
@@ -2,7 +2,7 @@
 #ifndef NETSDB_TREE_RESULT_H
 #define NETSDB_TREE_RESULT_H
 
-#define MAX_BLOCK_SIZE 275000
+#define MAX_BLOCK_SIZE 500000
```

Comment:
I didn't touch this as it has already changed but I will keep it in mind for future tests.